Test existing marqo works with typing-extensions 4.5.0:
```
> pip install typing-extensions==4.5.0
Collecting typing-extensions==4.5.0
  Downloading typing_extensions-4.5.0-py3-none-any.whl (27 kB)
Installing collected packages: typing-extensions
Successfully installed typing-extensions-4.5.0

> pip install marqo
Collecting marqo
  Downloading marqo-1.2.0-py3-none-any.whl (31 kB)
Collecting packaging
  Using cached packaging-23.1-py3-none-any.whl (48 kB)
Collecting requests
  Using cached requests-2.31.0-py3-none-any.whl (62 kB)
Collecting urllib3
  Downloading urllib3-2.0.4-py3-none-any.whl (123 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 123.9/123.9 KB 4.7 MB/s eta 0:00:00
Collecting pydantic<2.0.0
  Using cached pydantic-1.10.12-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (3.1 MB)
Requirement already satisfied: typing-extensions>=4.2.0 in ./venv_typing_extensions_450/lib/python3.10/site-packages (from pydantic<2.0.0->marqo) (4.5.0)
Collecting charset-normalizer<4,>=2
  Downloading charset_normalizer-3.2.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (201 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 201.8/201.8 KB 14.7 MB/s eta 0:00:00
Collecting certifi>=2017.4.17
  Downloading certifi-2023.7.22-py3-none-any.whl (158 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 158.3/158.3 KB 23.3 MB/s eta 0:00:00
Collecting idna<4,>=2.5
  Using cached idna-3.4-py3-none-any.whl (61 kB)
Installing collected packages: urllib3, pydantic, packaging, idna, charset-normalizer, certifi, requests, marqo
Successfully installed certifi-2023.7.22 charset-normalizer-3.2.0 idna-3.4 marqo-1.2.0 packaging-23.1 pydantic-1.10.12 requests-2.31.0 urllib3-2.0.4

> python3 
Python 3.10.12 (main, Jun 11 2023, 05:26:28) [GCC 11.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import marqo
>>> mq = marqo.Client()
>>> 
```

But not for 4.4.0:


```
> pip install typing-extensions==4.4.0
Collecting typing-extensions==4.4.0
  Downloading typing_extensions-4.4.0-py3-none-any.whl (26 kB)
Installing collected packages: typing-extensions
Successfully installed typing-extensions-4.4.0

> pip install marqo
Collecting marqo
  Using cached marqo-1.2.0-py3-none-any.whl (31 kB)
Collecting urllib3
  Using cached urllib3-2.0.4-py3-none-any.whl (123 kB)
Collecting packaging
  Using cached packaging-23.1-py3-none-any.whl (48 kB)
Collecting pydantic<2.0.0
  Using cached pydantic-1.10.12-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (3.1 MB)
Collecting requests
  Using cached requests-2.31.0-py3-none-any.whl (62 kB)
Requirement already satisfied: typing-extensions>=4.2.0 in ./venv_typing_extensions_440/lib/python3.10/site-packages (from pydantic<2.0.0->marqo) (4.4.0)
Collecting charset-normalizer<4,>=2
  Using cached charset_normalizer-3.2.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (201 kB)
Collecting certifi>=2017.4.17
  Using cached certifi-2023.7.22-py3-none-any.whl (158 kB)
Collecting idna<4,>=2.5
  Using cached idna-3.4-py3-none-any.whl (61 kB)
Installing collected packages: urllib3, pydantic, packaging, idna, charset-normalizer, certifi, requests, marqo
Successfully installed certifi-2023.7.22 charset-normalizer-3.2.0 idna-3.4 marqo-1.2.0 packaging-23.1 pydantic-1.10.12 requests-2.31.0 urllib3-2.0.4
(venv_typing_extensions_440)

> python3 
Python 3.10.12 (main, Jun 11 2023, 05:26:28) [GCC 11.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.

>>> import marqo
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/lib/python3.10/site-packages/marqo/__init__.py", line 1, in <module>
    from marqo.client import Client
  File "/lib/python3.10/site-packages/marqo/client.py", line 7, in <module>
    from typing_extensions import deprecated
ImportError: cannot import name 'deprecated' from 'typing_extensions' (/lib/python3.10/site-packages/typing_extensions.py)
```

--- a/setup.py
+++ b/setup.py
@@ -18,7 +18,7 @@ setup(
         "tox"
     ],
     name="marqo",
-    version="1.2.0",
+    version="1.2.1",
     author="marqo org",
     author_email="org@marqo.io",
     description="Tensor search for humans",

